karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
Hi @aka-jain @deshraj  This seems wrong. It should not be kept in the root, the gulpfile as well as the Karma.config must be inside the frontend folder.
The root is specific for the files that are common to `frontend` and `backend`.